stats/cloud/api.go

```diff
@@ -0,0 +1,94 @@
+package cloud
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/loadimpact/k6/stats"
+	"github.com/pkg/errors"
+)
+
+type sample struct {
+	Type   string     `json:"type"`
+	Metric string     `json:"metric"`
+	Data   sampleData `json:"data"`
+}
+
+type sampleData struct {
+	Type  stats.MetricType  `json:"type"`
+	Time  time.Time         `json:"time"`
+	Value float64           `json:"value"`
+	Tags  map[string]string `json:"tags,omitempty"`
+}
+
+type ThresholdResult map[string]map[string]bool
+
+type TestRun struct {
+	Name       string              `json:"name"`
+	ProjectID  int                 `json:"project_id,omitempty"`
+	Thresholds map[string][]string `json:"thresholds"`
+	// Duration of test in seconds. -1 for unknown length, 0 for continuous running.
+	Duration int64 `json:"duration"`
+}
+
+type CreateTestRunResponse struct {
+	ReferenceID string `json:"reference_id"`
+}
+
+func (c *Client) CreateTestRun(testRun *TestRun) (*CreateTestRunResponse, error) {
+	url := fmt.Sprintf("%s/tests", c.baseURL)
+	req, err := c.NewRequest("POST", url, testRun)
+	if err != nil {
+		return nil, err
+	}
+
+	var ctrr = CreateTestRunResponse{}
```

Comment:
`var` here is redundant; `ctrr := CreateTestRunResponse{}` or `var ctrr CreateTestRunResponse`